tools/replaydiff.cpp

```diff
@@ -0,0 +1,107 @@
+//////////////////////////////////////////////////////////////////////////////
+// Copyright (c) 2016-19, Lawrence Livermore National Security, LLC and Umpire
+// project contributors. See the COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (MIT)
+//////////////////////////////////////////////////////////////////////////////
+
+#include <iostream>
+#include <fstream>
+#include <ratio>
+#include <string>
+#include <vector>
+
+#include "umpire/tpl/cxxopts/include/cxxopts.hpp"
+#include "util/Replay.hpp"
+
+static std::vector<std::string> positional;
```

Comment:
Does positional really need to be a static global here?

Reply:
This is fixed